### PR TITLE
Add complete step by step installation guide

### PR DESCRIPTION
**Pull request by nickyfantasy**
* Add virtualenv and anaconda
* Remove overlap guide
* Clarify some confusing language

### CODE REVIEW

**Comment by nickyfantasy** [comment]
I will add a section of how to install and build from sources in next PR

File: README.md [hunk 1]
```diff
@@ -64,16 +64,94 @@ Histogram can be used to visualize parameter distribution and trends for any ten
...
 
 ```
-# Install the VisualDL. Preferably under a virtual environment.
+# Install the VisualDL. Preferably under a virtual environment or anaconda.
 pip install --upgrade visualdl
 
 # run a demo, vdl_create_scratch_log will create logs for testing.
 vdl_create_scratch_log
-visualDL --logdir=scratch_log --port=8080
+visualdl --logdir=scratch_log --port=8080
```
Comment (jetfuel) [inline README.md:72]: I am not sure if you need to change the upper case to lower case, It seems some system will default to lower case and some don't

**Review by jetfuel — APPROVED** [review]
LGTM

**Review by daming-lu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (daming-lu) [reply README.md:72]: Ubuntu is case sensitive

**Pull request merged**